core/src/main/java/org/infinispan/configuration/global/GlobalConfigurationBuilder.java

```diff
@@ -15,6 +15,8 @@
 
 public class GlobalConfigurationBuilder implements GlobalConfigurationChildBuilder {
 
+   public static final String ZERO_CAPACITY_NODE_FEATURE = "zero-capacity-node";
```

Comment:
I am not liking this here. Especially if this feature is removed earlier. I would prefer if this variable was defined on a class that isn't marked as public.

Reply:
I moved the constant to be private and if I need to access from somewhere else I will find another place